src/Core/src/Platform/iOS/MauiLabel.cs

```diff
@@ -49,7 +49,7 @@ public override void DrawText(RectangleF rect)
 		RectangleF AlignVertical(RectangleF rect)
 		{
 			var frameSize = Frame.Size;
-			var height = Lines == 1 ? Font.LineHeight : SizeThatFits(frameSize).Height;
+			var height = Lines == 1 ? Font.LineHeight : SizeThatFitsWithinInsets(frameSize).Height;
```

Comment:
If i understand this PR, this change is just so that this line gets the size without the min clamping?